transifex/api/jsonapi/resources.py

```diff
@@ -862,23 +863,32 @@ def __eq__(self, other):
         return self.as_resource_identifier() == other.as_resource_identifier()
 
     def __repr__(self):
+        r = reprlib.Repr()
+        r.maxlevel = 2
+        r.maxdict = 2
+        r.maxlist = 2
+        r.maxstring = 20
+
```

Comment:
Fwiw, this probably should be a module-level const variable, since it's immutable.